.github/workflows/ci.yml

```diff
@@ -241,13 +218,15 @@ jobs:
         timeout-minutes: 5
         env:
           TEST_NAME: ${{ matrix.test_name }}
-        run: ./scripts/tests/perf/${{ matrix.test_name }}.sh
+        run: |
+          ./scripts/tests/perf/${{ matrix.test_name }}.sh
+          jq '{ "${{ matrix.test_name }}": { "throughput": { "value": .end.sum_received.bits_per_second } } }' ./${{ matrix.test_name }}.json > ./${{ matrix.test_name }}.bmf.json
```

Comment:
Previously, we would track send and received metrics. I changed this to always use the "received" value for consistency and esp. in the case of UDP, I think the throughput _arriving_ is the best metric.